docs/rules.md

```diff
@@ -15,7 +15,7 @@ To update it edit `docs/rules.md` in the
 | :negative_squared_cross_mark: | warning | dangerous_contract | | Dangerous XPCOM contract ID |  | [testcases/javascript/actions.py](https://github.com/mozilla/amo-validator/blob/master/validator/testcases/javascript/actions.py)| | ('js', 'actions', 'dangerous_contract') | **Removed** |
 | :white_check_mark: | warning |  banned_identifier | | Banned or deprecated JavaScript Identifier | | | ('js', 'actions', 'banned_identifier') | BANNED_NEWTHREAD |
 | :white_check_mark: | warning |  banned_identifier | | Banned or deprecated JavaScript Identifier | | | ('js', 'actions', 'banned_identifier') | BANNED_PROCESSNEXTEVENT |
-| :x: | warning | complex_prefs_defaults_code | | Complex code should not appear in preference defaults files | | | | |
+| :white_check_mark: | warning | complex_prefs_defaults | | Complex code should not appear in preference defaults files | | [testcases/javascript/actions.py](https://github.com/mozilla/amo-validator/blob/master/validator/testcases/javascript/actions.py#L427) | ('testcases_javascript_actions', '_call_expression', 'complex_prefs_defaults_code')| COMPLEX_PREFS_DEFAULTS |
```

Comment:
Do we want to change our code name for this? "Complexity" is a misnomer here; we literally allow only two methods with very specific functionality :smile: 

Maybe our code should be `ONLY_PREFS_IN_DEFAULTS`